### PR TITLE
PD 390: [DR] ClusterPair doc is missing pointer for token

### PR DESCRIPTION
**Pull request by andreipope**
Signed-off-by: Andrei Popescu <andrei.popescu@gmail.com>

### CODE REVIEW

**Comment by andreipope** [comment]
@harsh-px @nathan-px Hey guys, please review this when you get the chance. Thanks!

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 1]
```diff
@@ -104,9 +104,14 @@ status:
   storageStatus: ""
 ```
 
-## Enable DR mode
+## Enable the disaster recovery mode
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: This one is tricky, but I think you should remove "the" here.

```suggestion
## Enable disaster recovery mode
```

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 2]
```diff
@@ -104,9 +104,14 @@ status:
...
 
-By default, every 7th migration will be a full migration. For DR though, we want every migration to be incremental. This can be enabled by setting the mode for the clusterPair to DisasterRecovery. For example, for the clusterPair below, we can add the mode under spec.options:
+You can enable the disaster recovery mode by specifying the following fields in the `options` section of your `ClusterPair`:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]:
```suggestion
You can enable disaster recovery mode by specifying the following fields in the `options` section of your `ClusterPair`:
```

**Comment by andreipope** [comment]
@harsh-px Hey Harsh, please review this when you get the chance. Thanks!

**Comment by andreipope** [comment]
Hey @harsh-px , please review this when you get the chance. Thanks!

**Pull request merged**